src/main/java/uk/gov/hmrc/example/ServiceConnector.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2015 HM Revenue & Customs
```

Comment:
the other java source files dont have a copyright notice (if we are going to keep this one we should fix the year)